packages/dashboard/src/components/pageStructure/index.js

```diff
@@ -164,6 +194,28 @@ export function LeftRail() {
     trackClick(evt, path.trackingEvent);
   }, []);
 
+  // See how many templates are new based on the current date
+  useEffect(() => {
+    let mounted = true;
+    getTemplateCreationDates().then((templateEntries) => {
+      if (mounted) {
+        const currentDate = toDate(new Date(), getOptions());
+        setNumNewTemplates(
+          templateEntries
+            .map(([, creationDate]) => toDate(creationDate, getOptions()))
+            .map((creationDate) =>
+              differenceInSeconds(currentDate, creationDate)
```

Comment:
Why not use the `differenceInDays` util instead?

Reply:
just didn't realize it was there. thanks for mentioning. will update